apps/core/templates/components/language_selector.html

```diff
@@ -13,7 +13,7 @@
                 {% get_language_info for translation.locale.language_code as lang %}
```

Comment:
Should we somehow limit the list of languages in the language selector to "actually" translated languages?
The long list of languages does not look too pretty right now...

<img width="437" alt="Screenshot 2022-10-20 at 17 14 30" src="https://user-images.githubusercontent.com/143557/197015637-1deb0268-da08-4f25-8217-e21db6acf1fa.png">

Reply:
This actually works once I start translating the subpages:

<img width="319" alt="Screenshot 2022-10-20 at 17 35 45" src="https://user-images.githubusercontent.com/143557/197019271-b1ea10dd-dbaa-4dba-995b-8843e747d34d.png">

So perhaps this is just some kind of artifact of the local installation having a homepage version generated in each language when I build the fixtures... ?

Reply:
Ah, yes I think we should limit the fixtures to only generate a subset of the available languages. Two or three, combined with `latest` and `4.1.x` should be enough.

Reply:
I've changed the `buildfixtures` management command to only build for `en`, `nl`, and `is-is` with versions `latest` and `4.1.x`.

Reply:
Why would we use `is-is` instead of `is`? AFAIK the Icelandic spoken on Iceland is 'the default'.

Reply:
Agree with @allcaps , we only use `is`

Reply:
That's what I thought, but Wagtail itself seems to use `is-is`: https://github.com/wagtail/wagtail/tree/main/wagtail/locales/locale

In that case, I guess I should update the others as well, e.g. `id-id`.

Reply:
Done, I've matched the codes to Django's `LANG_INFO`: https://github.com/django/django/blob/main/django/conf/locale/__init__.py

(except `pt` which I kept as `pt-pt` as it's clearer that way, and Django's `get_language_info` will handle this anyway)